eo-maven-plugin/src/test/java/org/eolang/maven/tojos/ForeignTojosTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven.tojos;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import org.eolang.maven.FakeMaven;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+/**
+ * Tests from {@link ForeignTojos}.
+ *
+ * @since 0.29.5
+ */
+final class ForeignTojosTest {
+
+    /**
+     * Testable foreign tojos.
+     */
+    private ForeignTojos tojos;
+
+    /**
+     * Set up environment before each test.
+     *
+     * @param tmp Temporary dir.
+     * @todo #2213:60min Create an instance of ForeignTojos in memory.
+     *  It would be better to be able to create an instance of ForeignTojos in
+     *  memory for test purposes.
+     */
+    @BeforeEach
+    void setUp(@TempDir final Path tmp) {
+        this.tojos = new ForeignTojos(() -> new FakeMaven(tmp).foreign());
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "abs",
+        "org.eolang.int",
+        "QQ.io.stdout"
+    })
+    void contains(final String name) {
+        this.tojos.add(name);
+        MatcherAssert.assertThat(
+            this.tojos.contains(name),
+            Matchers.is(true)
+        );
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "abs, sba",
+        "org.eolang.int, org.eolang.float",
+        "QQ.io.stdout, QQ.txt.sprintf"
+    })
+    void doesNotContain(final String name, final String check) {
+        this.tojos.add(name);
```

Comment:
@maxonfjvipon Do you need to add elements to `tojos` for that test? If you wan't to emphasise that `tojos` already has elements (tojos is not empty) and you are looking among them then I would recommend to init them inside `setUp` function.

Or if you want to leave the test as is, just give more meaningful names for the `name` and `check` variables (maybe something like `name` -> `existing` and `check` -> `considered`)

Reply:
@volodya-lombrozo I preferred more meangfull names